sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -177,3 +153,27 @@
 
 This package's
 [documentation](https://github.com/Azure/azure-sdk-for-java/blob/azure-core-amqp_1.0.0-preview.1/core/azure-core-amqp/README.md)
+## 1.0.0-beta.8 (12-03-2019)
```

Comment:
Parse date and either format it as by a common pattern or sort by datatime object.